src/rust/bitbox02/src/ui/ui.rs

```diff
@@ -166,6 +166,18 @@ pub fn screen_process() {
     }
 }
 
+pub fn screen_saver_disable() {
+    unsafe {
+        bitbox02_sys::screen_saver_disable();
+    }
+}
+
+pub fn screen_saver_enable() {
+    unsafe {
+        bitbox02_sys::screen_saver_enable();
+    }
+}
+
```

Comment:
please move these to a new file `bitbox02/src/screensaver.rs`, as the files here kind of mirror the C files/directories.

Reply:
moved it into that directory.

I named it `screen_saver.rs` since in the C code it's `screen_saver.c`. Is that ok?